packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -610,10 +610,11 @@ test.serial('writeGranules() writes a granule and marks as failed if any file wr
     granuleModel,
   });
 
-  t.true(await granuleModel.exists({ granuleId }));
+  const dynamoGranule = await granuleModel.get({ granuleId });
+  t.is(dynamoGranule.status, 'failed');
+  t.deepEqual(dynamoGranule.error.Error, 'Failed writing files to Postgres.');
 
   const pgGranule = await t.context.granulePgModel.get(knex, { granule_id: granuleId });
```

Comment:
Should be this to ensure we're looking up the unique granule for this test:

`const pgGranule = await t.context.granulePgModel.get(knex, { granule_id: granuleId, collection_cumulus_id: collectionCumulusId });`